src/nls/ru/strings.js

```diff
@@ -549,21 +549,39 @@ define({
     "COLOR_EDITOR_HSLA_BUTTON_TIP"              : "HSLa формат",
     "COLOR_EDITOR_USED_COLOR_TIP_SINGULAR"      : "{0} (Использовано {1} раз)",
     "COLOR_EDITOR_USED_COLOR_TIP_PLURAL"        : "{0} (Использовано {1} раза)",
-    
+
     // extensions/default/JavaScriptCodeHints
     "CMD_JUMPTO_DEFINITION"                     : "Перейти к определению",
     "CMD_SHOW_PARAMETER_HINT"                   : "Показывать подсказки по аргументам функции",
     "NO_ARGUMENTS"                              : "<нет аргументов>",
-    
+
     // extensions/default/JSLint
     "JSLINT_NAME"                               : "JSLint",
-    
+
     // extensions/default/QuickView
     "CMD_ENABLE_QUICK_VIEW"                     : "Быстрый просмотр при наведении",
-    
+
     // extensions/default/RecentProjects
     "CMD_TOGGLE_RECENT_PROJECTS"                : "Предыдущие проекты",
-    
+
     // extensions/default/WebPlatformDocs
-    "DOCS_MORE_LINK"                            : "Подробнее\u2026"
+    "DOCS_MORE_LINK"                            : "Подробнее\u2026",
+
+    //extensions/default/CodeFolding
+    "COLLAPSE_ALL"                  : "Свернуть все",
```

Comment:
As we are not using the preferences dialog, do we need these translations now? We could just use the root English string as it used only for setting the preferences variable.